src/components/SearchResults.js

```diff
@@ -0,0 +1,29 @@
+import propTypes from "prop-types";
+import "../styles/SearchResults.css";
+
+const SearchResults = ({ results }) => {
+  if (!results.length) {
+    return <p>Enter a valid search term!</p>;
+  } else {
+    return (
+      <div className="search-results">
+        {results.map((result, index) => {
+          return (
+            <img
+              src={result}
+              alt="space"
```

Comment:
As a suggestion for future improvements, could this alt tag be provided by data returned from the API?